demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -337,3 +342,69 @@ def parse_docker_image(self, docker_image):
                 return 'demisto/python', self.get_docker_image_latest_tag('demisto/python', None)
             else:
                 return 'demisto/python3', self.get_docker_image_latest_tag('demisto/python3', None)
+
+    @staticmethod
+    def get_docker_image_latest_tag_from_iron_bank_request(docker_image_name):
+        """
+        Get the latest tag for a docker image by request to Iron Bank Repo.
+        Args:
+            docker_image_name: The docker image name.
+
+        Returns:
+            The latest tag for the docker image.
+        """
+        project_name = docker_image_name.replace('demisto/', '')
+        api_url = 'https://repo1.dso.mil/api/v4/projects/dsop%2Fopensource%2Fpalo-alto-networks%2Fdemisto%2F'
+        commits_url = api_url + f'{project_name}/pipelines'
+        manifest_url = api_url + f'{project_name}/repository/files/hardening_manifest.yaml/raw'
+
+        try:
+            last_commit = DockerImageValidator._get_latest_commit(commits_url, docker_image_name)
+            if not last_commit:
+                return ''
+            manifest_file_content = DockerImageValidator._get_manifest_from_commit(manifest_url, last_commit)
+            if not manifest_file_content:
+                return ''
+        except Exception as e:
+            raise(e)
+
+        version_pattern = 'tags:\n- (.*)\n'
+        latest_version = re.findall(version_pattern, manifest_file_content)
+
+        # If manifest file does not contain the tag:
+        if not latest_version:
+            raise Exception('(Iron Bank) Manifest file does not contain tag in expected format.')
+
+        return latest_version[0].strip('"')
+
+    @staticmethod
+    def _get_manifest_from_commit(manifest_url, commit_id):
+        # gets the manifest file from the specified commit in Iron Bank:
+        res = requests.get(url=manifest_url, params={'ref': commit_id}, verify=False, timeout=TIMEOUT)
+
+        # If file does not exists in the last commit:
+        if res.status_code != 200:
+            raise Exception("Missing manifest file in the latest successful commit.")
+
+        return res.text
+
+    @staticmethod
+    def _get_latest_commit(commits_url, docker_image_name):
+        # Get latest commit in master which passed the pipeline of the project in Iron Bank:
+        res = requests.get(url=commits_url, params={'ref': 'master', 'status': 'success'}, verify=False,
+                           timeout=TIMEOUT)
```

Comment:
How do you handle pagination? See: https://docs.gitlab.com/ee/api/pipelines.html#pipelines-pagination